README.md

```diff
@@ -26,9 +26,9 @@
 
 ## Intro
 
-Export your excel VBA project source code for use with Git (or any VCS) from the Excel Developer ribbon.
+Export your excel VBA project source code for use with Git (or any VCS) from the Excel Developer ribbon, a pure VBA add-in for code export, no need to install a COM add-in
 
-For a while now I have used this code so that all the associated VBA files used in a VBA project (*.cls, *.bas, *.frm files) can be effortlessly exported for use with a Version Control System.
+Use this code so that all the associated VBA files used in a VBA project (*.cls, *.bas, *.frm files) can be effortlessly exported for use with a Version Control System.
```

Comment:
Nice change to the way this is framed. What do you think about "Use this Add-In" instead of "Use this code"?